tests/test_hooks/test_param_scheduler_hook.py

```diff
@@ -74,6 +76,72 @@ def test_after_epoch(self):
         scheduler2.step = Mock()
         scheduler2.by_epoch = True
         runner.param_schedulers = dict(key1=[scheduler1], key2=[scheduler2])
+        hook.after_train_iter(runner, 0)
         hook.after_train_epoch(runner)
         scheduler1.step.assert_called()
         scheduler2.step.assert_called()
+
+    def test_after_val_epoch(self):
+        metrics = dict(loss=1.0)
+
+        # mock super _ParamScheduler class
+        class MockParamScheduler(_ParamScheduler):
+
+            def __init__(self):
+                pass
+
+            def _get_value(self):
+                pass
+
+        # runner.param_schedulers is None
+        hook = ParamSchedulerHook()
+        runner = Mock()
+        runner.param_schedulers = None
+        assert hook.after_val_epoch(runner, metrics) is None
+
+        # metrics is None
+        scheduler = Mock()
+        runner.param_schedulers = [scheduler]
+        assert hook.after_val_epoch(runner, None) is None
+
+        # runner.param_schedulers is not built
+        hook = ParamSchedulerHook()
+        runner.param_schedulers = dict(key1=[None], key2=[None])
+        assert hook.after_val_epoch(runner) is None
```

Comment:
`hook.after_val_epoch` only returns None so it seems like this assert clause is not meaningful.